src/webview/static/base.css

```diff
@@ -404,42 +404,32 @@ body {
 
 /*
  Message contents.
+
+ For the body of a message, we use nearly the same HTML as the webapp does.
+ The styles therefore have a lot in common.  For the webapp's version of
+ these styles, compare static/styles/rendered_markdown.scss .
```

Comment:
.css, I think: https://github.com/zulip/zulip/commit/a3d26d701eafc8f4d2ad260764dee111225741c4

Reply:
Hmm indeed! Happened just this week, it looks like.